client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -103,9 +103,11 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
     };
   });
 
-  const manualTags = application?.tags?.filter((t) => t.source === "") ?? [];
+  const manualTags =
+    application?.tags?.filter((t) => t?.source ?? "" === "") ?? [];
 
-  const nonManualTags = application?.tags?.filter((t) => t.source !== "") ?? [];
+  const nonManualTags =
+    application?.tags?.filter((t) => t?.source ?? "" !== "") ?? [];
 
   // Allow all tags to be selected manually, even if they are included from another source
   const allowedManualTags = tags;
```

Comment:
```suggestion
  const allowedManualTags = tags.filter((tag) => !manualTags.includes(tag));
```

Thoughts on adding this to handle the dupe tag scenario discovered? Can totally be another PR. Just posting here before I forget.

Reply:
Is the dupe tag scenario still a problem?

IIRC, all tags should be selectable as a manual tag even if the same tag shows up with two different sources.  Is there yet another dupe tag scenario?

Reply:
Yeah - when importing an app with a bunch of tags already applied ( potentially duplicate tags), this was necessary. Open to revisiting this. I think the hub is adding additional duplication logic.

Reply:
So there can be tags on an application that don't exist as a tag?   That's a weird import behavior...